data.sql

```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name,date_of_birth,weight_kg,neutered,escape_attempts)
+VALUES ('Agumon','Feb 3, 2020',10.23,TRUE,0);
```

Comment:
- [ ] Kindly update date to `Feb 2` not `Feb 3`

Thank you 🙂

Reply:
![date](https://user-images.githubusercontent.com/75126481/134151710-e2f3c418-f943-429a-bda4-52129e95f221.png)

In the requirements, they wrote Feb 3rd that is why I put Feb 3rd not Feb 2